SocialCareCaseViewerApi.Tests/V1/Helpers/TestHelpers.cs

```diff
@@ -185,7 +186,7 @@ public static InfrastructurePerson CreatePerson(long? personId = null, string? f
                 .RuleFor(p => p.Nationality, f => f.Address.Country())
                 .RuleFor(p => p.NhsNumber, f => f.Random.Number(int.MaxValue))
                 .RuleFor(p => p.PersonIdLegacy, f => f.Random.String2(16))
-                .RuleFor(p => p.AgeContext, f => f.Random.String2(1))
+                .RuleFor(p => p.AgeContext, f => ageContext ?? f.Random.String2(1, "ACac"))
```

Comment:
Why do we use lower case `a` & `c`?

Reply:
It is probably quite nice if we are able to ignore the case for A or C from what the frontend sends.
In our actual code we .ToUpper I think.
So if our TestHelpers use either case it hopefully enforces that our code  is handling either case.

Reply:
I see. Thanks! 🙏🏾